php-exercise-2/controllers/calculatorController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+require_once 'model/CalculatorModel.php';
+
+class CalculatorController
```

Comment:
It is best practice to have one naming convention for your class name and file name.

Reply:
I made each of the files's names identical to the class name it contains.

Reply:
done